### DISCUSSION TITLE
Inventory commands

### DISCUSSION THREAD

**Discussion by TomCammil** [ORelio/Minecraft-Console-Client#2023] [Q&A]
Hi, im relatively new to console client. I am currently trying to make a script for a server where an account opens kit crates, detects what kit it gets, if it is a good kit, it stores it in /pv 1-5 etc. My problem here is that i wont be able to stash the kits in pvs with the default "/inventory 1 click 2". I need something that shiftclicks the item i specify so that it gets stashed directly.

I was browsing and i found tons of .cs folders where commands such as these seem to have been more developed, to me it looks like addons? I found one for my specific problem, where it looks like the option "ShiftClick" amongst many other clicktypes are added to the /inventory command. But i am confused as to how i apply these .cs files to work in my console client, if that even is what to do. Typing "ShiftClick" in the default /inventory command doesnt work atleast, and that is what i am hoping to make happen.

**Comment by ORelio** [comment]
In order to do more advanced actions than left/middle/right click I think you will need to use the C# API, which requires to program your own bot, basically. Sorry but the commands can't do everything at the moment.

**Reply by TomCammil** [reply]
Okay, so at the moment there is no way to transfer something from inventory to chest/pv with 1 click? Because from what I see right now, I am only able to move things from my inventory to a chest/pv by making the script pick it up and make it hover in my inventory, and then place it in a specified slot in the chest/pv. Would be really handy if I knew a way of just stashing it all quicker, without making a superlong script for filling a chest/pv.
